Formula/genstats.rb

```diff
@@ -1,8 +1,13 @@
 class Genstats < Formula
   desc "Generate statistics about stdin or textfiles"
-  homepage "https://www.vanheusden.com/genstats/"
-  url "https://www.vanheusden.com/genstats/genstats-1.2.tgz"
-  sha256 "f0fb9f29750cdaa85dba648709110c0bc80988dd6a98dd18a53169473aaa6ad3"
+  homepage "https://web.archive.org/web/20180831170016/www.vanheusden.com/genstats/"
+  url "https://web.archive.org/web/20150331055106if_/vanheusden.com/genstats/genstats-1.2.tgz"
+  sha256 "9988264357211a24f7024db05e24ed88db58227a626330114309147eb7078f6e"
+  license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-only"
```

Reading through https://web.archive.org/web/20150919193005/http://www.vanheusden.com/license.txt, I think it's safe to call this GPL2 only to remove the deprecated license tag.

Reply:
The license file itself isn't enough to determine that though. The source files will have to indicate if you can use later versions of the GPL.

Reply:
I'm not seeing anything in the source files allowing "Or later".